packages/cli/src/commands/init/init.ts

```diff
@@ -307,6 +309,27 @@ function userAgentPackageManager() {
   return null;
 }
 
+const createGitRepository = async (folder: string) => {
+  try {
+    await execa('git', ['init'], {cwd: folder});
+    await execa('git', ['branch', '-M', 'main'], {cwd: folder});
+    await execa('git', ['add', '.'], {cwd: folder});
+    await execa(
+      'git',
+      [
+        'commit',
+        '-m',
+        `Initial commit\n\nGenerated by ${packageJson.name} ${packageJson.version}.`,
```

Comment:
I’d avoid using package json info. Users don’t know which cli version they use, all they care about is RN version

Reply:
Agree, CLI version is not very informative. I don't know if we want to declare in codebase which CLI version points to which RN version (I know that there's idea of doing this in this PR https://github.com/react-native-community/cli/pull/1983/files). 🤔

Reply:
Maybe let's RN version directly from node_modules? 
Sth like `cat node_modules/react-native/package.json`

Reply:
https://github.com/react-native-community/cli/pull/2088/commits/a725e41e2adc27f1b51f861ffef27bf5ebd4e669 ✅